tests/functionality/scopflow/CMakeLists.txt

```diff
@@ -136,7 +136,7 @@ if(EXAGO_INSTALL_TESTS)
     COMMAND
     ${MPICMD}
     "-n"
-    3
+    1
```

Comment:
I am suspicious as this seems like an unnecessary change

Reply:
The name of this test is FUNCTIONALITY_TEST_SCOPFLOW_HIOP_SERIAL_TESTSUITE so it seems like it should be run on 1 process.